rhessys/tec/input_new_patch_mult.c

```diff
@@ -42,7 +42,7 @@
 #include "rhessys.h"
 #include "phys_constants.h"
 #define ONE 1.0
-
+#include "params.h"
 
  void  input_new_patch_mult(
```

Comment:
I mean, I'll delete the changes not related to the worldtag_read so that you can merge the 'Xiaoli' branch to 'develop' branch. 

For the duplicating issue in input_new_patch_mult.c, I'll talk with Naomi later and find a solution then.